tensorflow/python/kernel_tests/nn_ops/rnn_cell_test.py

```diff
@@ -38,6 +38,7 @@
 from tensorflow.python.ops import gradients_impl
 from tensorflow.python.ops import init_ops
 from tensorflow.python.ops import math_ops
+from tensorflow.python.ops import random_ops
```

Comment:
This dependency needs to be added in the build file:
tensorflow/python/kernel_tests/nn_ops/BUILD